framework/src/modules/chain/bft/bft.js

```diff
@@ -15,7 +15,10 @@
 'use strict';
 
 const EventEmitter = require('events');
-const { FinalityManager } = require('./finality_manager');
+const {
+	EVENT_BFT_FINALIZED_HEIGHT_CHANGED,
```

Comment:
One thing left, should not we export this event down from BFT module. So chain can utilize it.

Reply:
Good catch!